index.js

```diff
@@ -6,13 +6,41 @@ console.log("Here's your header:", header)
 /***** Deliverable 2 *****/
 
 
+header.style.color = "pink"
+
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+document.querySelector('h2').innerText = traveler.name 
+document.querySelector('div#profile em').innerText = traveler.nickname 
+document.querySelector('img').src = traveler.photo
+document.querySelector('img').alt = traveler.name
```

Comment:
I loved the way you chained your attributes when updating! Later on when you work with more tags and data it might be helpful to create variables and update them that way - but this is awesome! Nice work 👍